admin-guides/sprint-calls-prep.md

```diff
@@ -1,26 +1,39 @@
-# Prep for Sprint Calls
+# Prep for Weekly Calls
 
-## Who does this Prep
+## Who is Responsible
 
-Ideally, the lead of each call, or a specific person.
+- The host stays the same for each month.
+- On the last week of each month, identify a host for the following month.
+- If the host can’t make it some week, they are responsible for delegating to someone and making sure they have permissions at least a day ahead.
 
-## Steps to Take
-
-**Advanced preparation**
+## Host Responsibilities
 
 On Wednesday of the preceding week:
 - Make sure that [node-github-issue-bot](https://github.com/ipfs/node-github-issue-bot) has created the next sprint issue. It should use templates from [here](https://github.com/ipfs/node-github-issue-bot/tree/master/templates). If it breaks, look at [the README in ipfs/infrastructure](https://github.com/ipfs/infrastructure/tree/master/sprintbot).
 - Manually fill out the Notes link with [the templates](https://github.com/ipfs/pm/tree/master/templates). To not do this manually, follow [this issue](https://github.com/ipfs/node-github-issue-bot/issues/8).
 
-By Friday of the preceding week:
-- Adjust the calendar times to match the Sprint issue. To not manually do this, follow [this issue](https://github.com/ipfs/node-github-issue-bot/issues/4).
+#### Before the Call
+
+1. Make sure agenda is posted beforehand in GitHub using the current format (see #647 for an example). Ideally, this should be a day or more early so people can add items before the meeting. (Also consider proposed changes in #636.)
+1. Post a notice just prior to and at the meeting time on IRC (#ipfs & #ipfs-dev), Twitter (@ipfsbot), & relevant channels.
 
-**Before the Call**  
+#### During the Call
 
-1. Launch a Zoom call and share the URL with [@dignifiedquire](https://github.com/dignifiedquire) or [@kubuxu](https://github.com/Kubuxu).
-1. Announce the call in IRC using `sprinthelper: announce`. To do this, manually copy over the name of the call, the number of the issue, the notes URL, the zoom call URL, and the Stream URL (which should be gotten from dig or Kuba). To automate this, close [this issue](https://github.com/ipfs/sprint-helper/issues/17).
+1. Be on Zoom promptly at the start time. You don’t need to moderate, but you have all the rights so you must be there to grant them.
```

Comment:
I think we [decided the host is the moderator](https://github.com/ipfs/pm/issues/644#issuecomment-396632064), so we should get rid of the second sentence here, right?